### PR TITLE
Add jaeger integration docs

### PR DESCRIPTION
**Pull request by mkorolyov**
ScyllaDB is suitable and verified as storage backend for the Jaeger Server while it is not supported by Jaeger Server maintainers, there is a working demo example that runs smoothly. 

This PR adding a note that this integration exists and describing benefits of this setup with links to the jaeger itself along with the demo example.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -32,3 +32,4 @@ compile_commands.json
 .ccls-cache/
 .mypy_cache
 .envrc
+.idea
```
Comment (denesb) [inline .gitignore:35]: Please don't mix other changes with docs changes.

Comment (mkorolyov) [reply .gitignore]: reverted, thanks

**Comment by guy9** [comment]
Thanks @mkorolyov . @annastuchlik, can you review it?

**Review by annastuchlik — APPROVED** [review]
Looks great.

**Comment by annastuchlik** [comment]
@denesb Could you re-review and merge it?

**Review by denesb — APPROVED** [review]
@mkorolyov before I can merge this PR, please sign our CLA, see https://github.com/scylladb/scylladb/blob/master/CONTRIBUTING.md#contributing-code-to-scylla.

**Comment by mkorolyov** [comment]
> @mkorolyov before I can merge this PR, please sign our CLA, see https://github.com/scylladb/scylladb/blob/master/CONTRIBUTING.md#contributing-code-to-scylla.

Done. Thanks.

**Comment by denesb** [comment]
Thanks @mkorolyov, can you please squash the commits into a single one and add the `doc:` prefix to the commit message? Thanks.

**Comment by denesb** [comment]
Queued (I removed accidental removal of new-line at end of `.gitignore`).

**Pull request closed without merging**